pkg/uploader/kopia/snapshot.go

```diff
@@ -88,10 +88,15 @@ func setupDefaultPolicy() *policy.Tree {
 
 // Backup backup specific sourcePath and update progress
 func Backup(ctx context.Context, fsUploader SnapshotUploader, repoWriter repo.RepositoryWriter, sourcePath string, realSource string,
-	forceFull bool, parentSnapshot string, tags map[string]string, log logrus.FieldLogger) (*uploader.SnapshotInfo, bool, error) {
+	forceFull bool, parentSnapshot string, volMode uploader.PersistentVolumeMode, tags map[string]string, log logrus.FieldLogger) (*uploader.SnapshotInfo, bool, error) {
 	if fsUploader == nil {
 		return nil, false, errors.New("get empty kopia uploader")
 	}
+
+	if volMode == uploader.PersistentVolumeBlock {
+		return nil, false, errors.New("unable to handle block storage")
```

Comment:
Would it be interesting to create a `const` with the error message? (this one is different from the other ones, but we could standardize, right?)

And would be good to be more explicit in the message, like
**"This version of Velero does NOT support block mode, it will be implemented in the future"** ?
(my fear is users opening issues not understanding the problem, instead of just asking when this feature will be available)

Reply:
@mateusoliveira43 Actually, your suggested wording change points out why we need different messages here. Kopia will add it in the future, but we will never support this in Restic.

Reply:
Resolving because as Scott said we want them to be different